velox/exec/OutputBuffer.h

```diff
@@ -168,22 +168,34 @@ class OutputBuffer {
         bool _noMoreBuffers,
         bool _noMoreData,
         bool _finished,
+        int64_t _bufferedBytes,
+        int64_t _bufferedPages,
+        int64_t _totalRowsSent,
+        int64_t _totalPagesSent,
         const std::vector<DestinationBuffer::Stats>& _buffersStats)
         : kind(_kind),
           noMoreBuffers(_noMoreBuffers),
           noMoreData(_noMoreData),
           finished(_finished),
+          bufferedBytes(_bufferedBytes),
+          bufferedPages(_bufferedPages),
+          totalRowsSent(_totalRowsSent),
+          totalPagesSent(_totalPagesSent),
           buffersStats(_buffersStats) {}
 
-    const core::PartitionedOutputNode::Kind kind;
+    core::PartitionedOutputNode::Kind kind;
 
     /// States of the OutputBuffer.
-    const bool noMoreBuffers;
-    const bool noMoreData;
-    const bool finished;
+    bool noMoreBuffers{false};
+    bool noMoreData{false};
+    bool finished{false};
+    int64_t bufferedBytes{0};
```

Comment:
Are these sums of the buffersStats? Would you be nice to add a comment.

Reply:
Yes, commented.